runtime/compiler/control/CompilationThread.cpp

```diff
@@ -7724,6 +7732,20 @@ TR::CompilationInfoPerThreadBase::postCompilationTasks(J9VMThread * vmThread,
    _vm = NULL;
    _addToJProfilingQueue = false;
 
+   if (_compInfo.getPersistentInfo()->getRemoteCompilationMode() == JITServer::SERVER)
```

Comment:
We need the `#if defined(J9VM_OPT_JITSERVER)` here. This is why the linter job fails